lua/theprimeagen/lazy/colors.lua

```diff
@@ -36,6 +36,9 @@ return {
         config = function()
             require('rose-pine').setup({
                 disable_background = true,
```

Comment:
```suggestion
```
I believe `disable_background` is deprecated according to the changelog here: https://github.com/rose-pine/neovim/releases/tag/v2.0.0

Reply:
You're right, I was just too lazy to check if that was the case when making my PR